metals/src/main/scala/scala/meta/internal/metals/Doctor.scala

```diff
@@ -45,7 +45,8 @@ final class Doctor(
       workspace,
       mtagsResolver,
       currentBuildServer,
-      javaHome
+      javaHome,
+      () => clientConfig.isCompletionItemResolve()
```

Comment:
This seems like the wrong config, no?

Reply:
wow, I don't have any idea why I did this 😳